README.md

```diff
@@ -29,16 +29,17 @@ post](https://coreos.com/blog/the-prometheus-operator.html).
 
 ## Prometheus Operator vs. kube-prometheus vs. community helm chart
 
-The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
+Prometheus Operator - The Prometheus Operator uses Kubernetes [custom resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to simplifiy the deployment and configuration of Prometheus, Alertmanager, and related monitoring components.
 
-[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
+kube-prometheus - [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) provides example configurations for a complete cluster monitoring
 stack based on Prometheus and the Prometheus Operator.  This includes deployment of multiple Prometheus and Alertmanager instances,
 metrics exporters such as the node_exporter for gathering node metrics, scrape target configuration linking Prometheus to various
 metrics endpoints, and example alerting rules for notification of potential issues in the cluster.
 
-The [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
+prometheus-operator - The [stable/prometheus-operator](https://github.com/helm/charts/tree/master/stable/prometheus-operator)
```

Comment:
This was moved to https://github.com/prometheus-community/helm-charts maybe we can point to that instead? Thanks!

Reply:
yes that woud be good,